Blocstagram/BLCImagesTableViewController.m

```diff
@@ -88,7 +88,16 @@ - (CGFloat) tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexP
 
 }
 
+- (void) tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    if (editingStyle == UITableViewCellEditingStyleDelete) {
+        [self.images removeObjectAtIndex:indexPath.row];
+        [tableView reloadData];
```

Comment:
I've updated and committed a new version.

On Thu, Apr 16, 2015 at 3:14 PM, Falko Buttler <notifications@github.com>
wrote:

> In Blocstagram/BLCImagesTableViewController.m
> <https://github.com/rmtreks1/BlocStagram/pull/1#discussion_r28484362>:
>
> > @@ -88,7 +88,16 @@ - (CGFloat) tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexP
> >
> >  }
> >
> > +- (void) tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath {
> > +
> > +    if (editingStyle == UITableViewCellEditingStyleDelete) {
> > +        [self.images removeObjectAtIndex:indexPath.row];
> > +        [tableView reloadData];
>
> Ideally, you should not call reloadData but tell the tableView to just
> delete this one cell. Check the documentation in UITableView for details.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rmtreks1/BlocStagram/pull/1/files#r28484362>.
>